kernel/Dockerfile.bcc

```diff
@@ -0,0 +1,105 @@
+ARG IMAGE
+FROM ${IMAGE} as ksrc
+
+FROM linuxkit/alpine:8a89682421abf0d886d777235a05f4a04a736df2 AS build
+RUN apk update && apk upgrade -a && \
+  apk add --no-cache \
+  argp-standalone \
+  autoconf \
+  automake \
+  bison \
+  build-base \
+  clang \
+  clang-dev \
+  clang-static \
+  cmake \
+  curl \
+  flex-dev \
+  fts-dev \
+  gettext-dev \
+  git \
+  iperf3 \
+  libedit-dev \
+  libtool \
+  llvm \
+  llvm-dev \
+  llvm-static \
+  luajit-dev \
+  m4 \
+  python \
+  zlib-dev \
+  && true
+
+RUN ln -s /usr/lib/cmake/llvm5/ /usr/lib/cmake/llvm; \
+    ln -s /usr/include/llvm5/llvm-c/ /usr/include/llvm-c; \
+    ln -s /usr/include/llvm5/llvm/ /usr/include/llvm
+
+WORKDIR /build
+COPY ./bcc.patches/ ./
+COPY Dockerfile.bcc /
+COPY ./bcc.patches/error.h /usr/include/
```

Comment:
you already have `error.h` and `cdefs.h` in the build context. Why not copy them with `RUN cp` instead (or better move them with `mv`)?